app/views/shared/_footer.html.erb

```diff
@@ -1,35 +1,36 @@
-  <div class="secondary-footer row text-center">
-  <div class="col-md-2 col-xs-4">
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-  </div>
-  <div class="col-md-2 col-xs-4">
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-  </div>
-  <div class="col-md-2">
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-    <div class="footer-navlink"><a href="https://github.com/samvera/hyku">NavLink</a></div>
-  </div>
-  <div class="col-md-6">
-    <div class="footer-logo">
-      <a href="<%= root_path %>">
-        <%= image_tag "ADL-footer-logo.png", :class => "logo-img" %>
-      </a>
+<div class="secondary-footer">
```

Comment:
Reading notes: I see we're removing `.text-center` from this div; and not moving it down.

Reply:
yes- the text will be centered only at a lower breakpoint, which i needed to add in the custom stylesheet. there aren't responsive BS classes for text alignment in bootstrap 3.4 unfortunately